user/helm-chart/values-kubernets.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 The KubeStellar Authors.
```

Comment:
Typo in this file's name: "kubernetes" vs "kubernets"

Reply:
thank you